package.json

```diff
@@ -68,14 +68,15 @@
         "@emotion/core": "^10.0.9",
         "@emotion/styled": "^10.0.9",
         "@mdx-js/loader": "^0.17.5",
-        "@mdx-js/mdx": "^0.17.5",
+        "@mdx-js/mdx": "^0.15.0",
```

Comment:
Same question here, why did we need to downgrade this package?

Reply:
Next had some peerdep mismatches if we use later versions, and we wanted to avoid adding errors on installing our stuff.

Reply:
yes , `zeit/next-mdx` has dependency to the downgraded version